model.py

```diff
@@ -5362,39 +5525,33 @@ def consolidate_works(cls, _db, calculate_work_even_if_no_author=False,
 
 
     def calculate_work(self, even_if_no_author=False, known_edition=None):
```

Comment:
known_edition has started to cause problems. I think with the advent of presentation_edition it's no longer necessary. There used to be situations surrounding OPDS import where the LicensePool's presentation edition was missing and had to be passed in, but now I think we can consistently rely on LicensePool.presentation_edition working properly.

I'm not changing the use of known_edition in this branch but hopefully this explains a couple changes I made later on.